x-pack/plugins/cases/public/get_create_case.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { lazy, Suspense } from 'react';
+import { CreateCaseProps } from './components/create';
+
+export const getCreateCaseLazy = (props: CreateCaseProps) => {
+  const CreateCaseLazy = lazy(() => import('./components/create'));
+  return (
+    <Suspense fallback={null}>
```

Comment:
Why `null` as a fallback? Shouldn't we show a loading spinner?

Reply:
copied from actions, ill change this to have the spinner